.gitignore

```diff
@@ -0,0 +1,11 @@
+# package directories
+node_modules
+jspm_packages
+
+# Serverless directories
+.serverless
+serverless.yml
```

Comment:
I think file serverless.yml should not be in gitignore.
You can try to use the serverless-dotenv-plugin package to avoid adding environment variables in the serverless.yml file.

Reply:
Agree, will update in the next task